bungeecord/src/main/java/com/vexsoftware/votifier/bungee/events/VotifierEvent.java

```diff
@@ -1,23 +1,38 @@
 package com.vexsoftware.votifier.bungee.events;
 
 import com.vexsoftware.votifier.model.Vote;
-import net.md_5.bungee.api.plugin.Event;
+import net.md_5.bungee.api.Callback;
+import net.md_5.bungee.api.event.AsyncEvent;
+import net.md_5.bungee.api.plugin.Cancellable;
 
 /**
  * {@code VotifierEvent} is a custom BungeeCord event class that is sent
  * asynchronously allowing other plugins to listen for votes.
  */
-public class VotifierEvent extends Event {
+public class VotifierEvent extends AsyncEvent<VotifierEvent> implements Cancellable {
```

Comment:
Thumbs down to making this an `AsyncEvent`, it's a completely broken programming model along with being a breaking change.

I can see some value in having a cancelable event.

Reply:
I reverted the changes from the AsyncEvent but kept the Cancellable stuff.